.github/workflows/gradle.yml

```diff
@@ -89,7 +92,8 @@ jobs:
       if: matrix.e2e-tests == 'android'
       uses: reactivecircus/android-emulator-runner@v2
       with:
-        script: ./gradlew e2eAndroidTest -PisCI -Pselenium.version=$latest_snapshot
+        script: ./gradlew e2eAndroidTest -PisCI -Pselenium.version=$latest_snapshot  &&
```

Comment:
I would prefer to separate flutter tests into its own job and add it to the matrix above

Reply:
Got it. I'll update the pipline with new jobs for flutter tests.